kafka-rest/src/main/java/io/confluent/kafkarest/v2/KafkaConsumerReadTask.java

```diff
@@ -93,14 +82,11 @@ public boolean doPartialRead() {
     try {
       // Initial setup requires locking, which must be done on this thread.
       if (messages == null) {
-        parent.startRead();
```

Comment:
Seems `doPartialRead` should have a void return instead of boolean. The javadoc comment doesn't actually match the implemented behavior (even in the old code) and we just use `isDone` to check for backoff anyway.